src/executors/code-blocker/index.test.ts

```diff
@@ -0,0 +1,13 @@
+describe('code-blocker', () => {
```

Comment:
```suggestion
describe('code-blocker()', () => {
```

Reply:
This executor does not export as function